Engine/EditorUI/Panel/PopupsPanel/PanelPopupSceneManagement.cpp

```diff
@@ -2,109 +2,195 @@
 
 #include "Main/Application.h"
 #include "Module/ModuleEditor.h"
+#include "Module/ModuleFileSystem.h"
 #include "Module/ModuleScene.h"
 
+#include <algorithm>    // std::find
 #include <Brofiler/Brofiler.h>
+#include <imgui_internal.h>
+#include <imgui_stdlib.h>
 
 
 PanelPopupSceneManagement::PanelPopupSceneManagement()
 {
 	opened = false;
 	enabled = true;
 
-	save_scene_popup = ImGui::FileBrowser(ImGuiFileBrowserFlags_EnterNewFilename);
-	load_scene_popup = ImGui::FileBrowser();
-
 	window_name = "Popups Scene Management";
 }
 
 void PanelPopupSceneManagement::Render()
 {
-	RenderSaveScene();
-	RenderLoadScene();
-}
-
-void PanelPopupSceneManagement::RenderLoadScene()
-{
-	if (load_scene_shown)
+	if (popup_shown)
 	{
-		load_scene_shown = false;
-		load_scene_popup.Open();
+		popup_shown = false;
+		opened = true;
```

Comment:
Why you don't have here the same condition as in the file "PanelPopupResourceSelector.cpp"? after this with a 
if (!opened) return;

Reply:
This is not finished yet, so it will change at the future